PythonPonies/bangazon-api
# removed pycache folder and attempted a perm fix

# Description
Add description about what this pull request covers.
This should remove the pycache folder and prevent it from being repopulated with subsequent pushes to the project. 

## Number of Fixes
Type of number of fixes included in this PR.

Two: one immediate short term fix to delete the pycache folder and an attempt at a permanent fix by adding *.pyo to the .gitignore file as well as __pycache__/ both as separate line items. 

## Related Ticket(s)
List related ticket(s) that apply to the work done in this pull request
 #87  Remove pycache

## Problem to Solve
Describe the problem(s) this pull request solves.
This should eliminate the pycache folder from the final project and also prevent any future inadvertant pushes of the same file. It should also eliminate the need to git stash before changing branches locally. 

## Proposed Changes
Describe the proposed changes.
1. Delete the pycache folder.
2. Add *.pyo and __pycache__/ to the .gitignore file. 

## Expected Behavior
Describe the expected behavior.
This should remove the pycache folder. It should also prevent the same from every being pushed up again. 
## Steps to Test Solution

1. Git fetch this branch
2. Git checkout and open sublime. 
3. You should not see any pycache folder.
4. Change something small in any file (a comma, period, etc BUT BE SURE TO CHANGE IT BACK!)
5. Run git status
6. You should not see pycache files being tracked. 
7. There is no need to push your changes to this branch. Just delete the branch locally. 

## Testing

[ ] There are new unit tests in this PR, and I verify that there is full coverage of all new code.
[ ] I certify that all existing tests pass

## Documentation
[ ] There is new documentation in this pull request that must be reviewed..
[ ] I added documentation for any new classes/methods

## Deploy Notes
Notes regarding deployment the contained body of work.  These should note any
db migrations, etc.
